new-toast-site/fetch-mdx-post-files.js

```diff
@@ -1,41 +1,43 @@
-const fs = require("fs").promises;
-const fsReg = require("fs");
-const slugify = require("@sindresorhus/slugify");
-const mdx = require("@mdx-js/mdx");
-const util = require("util");
-const vm = require("vm");
-const rehypePrism = require("./rehype-prism-mdx");
-const rehypeSlug = require("rehype-slug");
-const rehypeLink = require("rehype-autolink-headings");
-const parse = require("rehype-parse");
-const unified = require("unified");
-const globby = require("globby");
-const chalk = require("chalk");
+import { promises as fs } from "fs";
```

Comment:
the toast.js and related files are all ES Modules based now, so I've updated imports and cleaned this file up a bit